### PR TITLE
TSLint

### PR DESCRIPTION
**Pull request by t-sauer**
Added `ember-cli-tslint` to the blueprint and adjusted the code/config to pass the tests after initialisation.

### CODE REVIEW

File: files/src/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import App from './main';
 import { ComponentManager, setPropertyDidChange } from '@glimmer/component';
+import App from './main';
```
Comment (rwjblue) [inline files/src/index.ts]: Which rule is forcing this change? I don’t fully understand why the various import statements are moved around...

Comment (t-sauer) [reply files/src/index.ts]: This change is caused by [`ordered-imports`](https://palantir.github.io/tslint/rules/ordered-imports/) which is included in the recommended preset.

Comment (locks) [reply files/src/index.ts]: @rwjblue do you prefer to disable the rule or keep the changes? I'd go with the changes.

Comment (rwjblue) [reply files/src/index.ts]: I’m not a giant fan of this specific rule, but meh.

Comment (t-sauer) [reply files/src/index.ts]: Sadly the recommended preset comes with too much opinions about stylistic choices in my opinion. I prefer what ESLint is doing in their recommended preset (not including any stylistic choices at all) but as far as I know, no similar preset exists for TSLint.

**Pull request merged**